server/repository/migrations/sqlite/2022-04-27T13-05_create_document_trigger/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TRIGGER document_insert_trigger
+  AFTER INSERT ON document
+  BEGIN
+    INSERT INTO changelog (table_name, record_id, row_action)
```

Comment:
add is_sync_update?

Reply:
thanks there is a second PR on top of this one coming and will add this at the end!